text/0000-long-error-codes-explanation-normalization.md

```diff
@@ -0,0 +1,143 @@
+
+Start Date: 2016-01-04
+
+RFC PR:
+
+Rust Issue: N/A
+
+# Summary
+
+Long error codes explanations haven't been normalized yet. For example, actually, not all error codes explanation provide an erroneous code example or a way to fix the error. This RFC intends to clarify this by providing a template to follow.
+
+# Motivation
+
+Long error codes explanations are a very important part of Rust. Having an explanation of what failed helps to understand the error and is appreciated by Rust developers of all skill levels. Providing an unified template is needed in order to help people who would want to write ones as well as people who read them.
+
+# Detailed design
+
+Here is what I propose:
+
+## Error description
+
+Giving a little more detailed error message. For example, with the following code:
```

Comment:
"Giving a little more detailed error message." -> "Provide a more detailed error message"

Using "code" is confusing here since we're talking about both source code and error codes, maybe change "code" on this line to "example".